kartothek/io_components/metapartition.py

```diff
@@ -1615,6 +1615,13 @@ def parse_input_to_metapartition(
         if len(obj) == 0:
             return MetaPartition(label=None, metadata_version=metadata_version)
         first_element = obj[0]
+        if isinstance(first_element, tuple):
```

Comment:
Please document the change in the Docstrings above.

Reply:
Added docstring documentation